src/common/webrequest_curl.cpp

```diff
@@ -165,6 +276,14 @@ wxWebRequestCURL::wxWebRequestCURL(wxWebSession & session,
     m_sessionImpl(sessionImpl)
 {
     m_headerList = NULL;
+    m_cancelPending = false;
+    m_hasInternalFail = false;
+
+#ifdef __WINDOWS__
+    m_devNull.Open("NUL","w");
+#else
+    m_devNull.Open("/dev/null","w");
+#endif
```

Comment:
I wonder if we should add a function returning the appropriate name to use to `wxFileName`.

This can/could be done later, of course, it's not really related to this PR.

Reply:
I just remove this since I now know I dont need to use CURLOPT_STDERR.